atd-vze/src/views/Crashes/PeopleDetailsCard.js

```diff
@@ -61,6 +61,7 @@ const PeopleDetailsCard = ({ data, refetch, ...props }) => {
             fieldConfig={primaryPersonDataMap[0]}
             data={primaryPersonData}
             sortField={"unit_nbr"}
+            secondSortField={"id"}
```

Comment:
fixes a bug where if the peoples card had multiple people in it, every time you edited a person the order would change which was really jarring bc we were only sorting on `unit_nbr`. now we sort first by `unit_nbr` and then person id